src/grt/src/GlobalRouter.cpp

```diff
@@ -151,13 +150,13 @@ std::vector<Net*> GlobalRouter::startFastRoute(int minRoutingLayer,
     setSelectedMetal(maxRoutingLayer);
   }
 
-  if (_pdRevForHighFanout != -1) {
-    _fastRoute->setAlpha(_alpha);
+  if (pdrev_min_fanout_ != -1) {
```

Comment:
it seems harmless to set alpha unconditionally because min_fanout control its use